indico/modules/events/persons/templates/management/person_list.html

```diff
@@ -100,6 +99,21 @@
                     </div>
                 {% endif %}
                 <div class="group">
+                    <div
+                        id="invitation-email-all"
+                        data-event-id="{{ event.id }}"
+                        data-person-selector="#event-participants-list tr[data-no-account] input[name=person_id][type=checkbox]"
+                        data-no-account="1">
```

Comment:
No need to put a value on boolean options; just put `data-no-account` and then check for `.dataset.noAccount !== undefined`